### DISCUSSION TITLE
Design Apple

### DISCUSSION THREAD

**Discussion by Libertycrash91** [KaoruKanon/homeassistant-config#3] [General]
Bonjour Kaoru,
Whaaouuu ton thème ainsi que tes cartes sont géniales. 
Je ne sais pas si tu auras le temps de répondre à mes questions mais je tente quand même. 
Je débute vraiment dans HA, j'ai découvert plein de chose sur le forum dans lequel je t'ai découvert. 
Vraiment bravo pour ton travail. 
J'aimerais avec le même style à ma manière bien sur. Mais utilisant HomeKit, j'adore ton Dashboard. 
Alors comment en explication simplifiée mais clair je pourrais arriver à ce style. 
Faut il un thème ?
Faut il des cartes particulières ?

**Comment by KaoruKanon** [comment]
Bonjour @Libertycrash91 

Hormis les fonts d'Apple qui sont copyrightées, tu as tout ce qu'il faut sur mon projet git

Faut tout coller ma conf dans ton dossier de conf de ton HA. Tu vas te retrouver à debug le dashboard parce que c'est mes équipements qui n'existe pas sur ton HA. Bien sûr, tu peux connecter tout ton système homekit a ce dashboard. 

Cependant, faut pas oublier d'activer le thème dans ton profil. Sinon, tu auras pas exactement le meme dashboard que moi

**Reply by Libertycrash91** [reply]
Hum, j'essaie de comprendre. La copie du fichier conf... c'est lequel !? 😅

**Reply by KaoruKanon** [reply]
C'est à dire tous les fichiers présents sur https://github.com/KaoruKanon/homeassistant-config

**Reply by Libertycrash91** [reply]
Vraiment cool, merci cela fonctionne. 
J'ai encore beaucoup de chose à changer. ( mes capteurs,etc. )
Le thème est vraiment génial. 

Question:
- Je n'arrive à faire aucun popup.. serais-tu quelle intégration ou autre tu Utilises ?
- je n'ai rien dans le slide bar non plus. 
- faut il faire quelque chose en particulier pour intégrer PS5 ou juste le python suffit ?

Désolé pour ces questions...
